packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_fr_FR.json

```diff
@@ -39,5 +39,6 @@
   "pci_projects_project_training_jobs_submit_data_permission_RO": "Lecture Seule",
   "pci_projects_project_training_jobs_submit_data_permission_RW": "Lecture & Ecriture",
   "pci_projects_project_training_jobs_submit_ovhai_info": "La commande suivante vous permet de lancer le même job grâce à notre CLI ovhai :",
-  "pci_projects_project_training_jobs_submit_ovhai_help": "Pour plus d'information sur les paramètres disponibles :"
+  "pci_projects_project_training_jobs_submit_ovhai_help": "Pour plus d'information sur les paramètres disponibles :",
+  "pci_projects_project_training_jobs_submit_select_data_select": "Je souhaite associer des volumes de données au travail"
```

Comment:
To confirmed but I'm not sure that `job` (translated here into` travail` in French) should be the case.

If we pay attention to existing translations from this file, I notice that we keep using `job`.

Could you please confirm if we can use `job` instead of `travail` here?
Thank you!

Reply:
Done